dockerfiles/base/all_tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+# Copyright (c) 2012-2017 Red Hat, Inc
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Marian Labuda - Initial Implementation
+
+# Build test image
```

Comment:
the build of the image will be done as well by CI job.
so user won't need to build image locally as well